css/components/sections.css

```diff
@@ -0,0 +1,98 @@
+#skills
+{
+    height: 85%;
+} 
+
+.section
+{
+    padding-left: 16%;
+    padding-right: 16%;
+
+    padding-top:50px;
+    height: 80%;
+}
+
+
+section:nth-child(EVEN) 
+{
+    background-color: rgb(228, 228, 228);
+}
+
+#content-container > .description 
+{
+    padding-bottom: 50px;
+    font-size: 16px;
+}
+
+
+.top-section
+{
+    padding-top: 0px;
+    height: 95%;
+
+    background-image: url('../../resources/earth-image-plain.png');
+    background-size: cover;
+    background-repeat: no-repeat;
+
+    z-index: 1;
+}
+
+#contact-me
+{
+    background-color: rgb(21, 62, 119);
+}
+
+
+#set-header > h3 
```

Comment:
I would say that ids are being used too often. They really should be quite limited.

Reply:
Yeah, I agree @ivanmora. I definitely see how I could have made the styles more generic. The sections could be a reusable class based on the size that I want them. Could have even made the background colors to be something more theme based; giving themed class properties to style particular divisions. Thank you for the feedback!